### PR TITLE
(DOCS-8321)(Workflows) Trigger changes

### PR DESCRIPTION
**Pull request by hestonhoffman**
### Merge instructions

Hold off on merging please

### CODE REVIEW

File: content/en/security/application_security/threats/security_signals.md [hunk 1]
```diff
@@ -59,8 +59,9 @@ Use [Incident Management][4] to create an incident for a security signal.
 
 Use [Workflow Automation][5] to manually trigger a workflow for a security signal. 
 
+1. Make sure the workflow you want to run has a security trigger.
 1. On the [Signals Explorer][2] page, select a security signal.
-2. Scroll down to the **What is Workflow Automation** section. 
+2. Scroll down to the **What is Workflow Automation** section.
 3. Click **Run Workflow**.
 4. On the workflow modal, select the workflow you want to run. Depending on the workflow, you may be required to enter additional input parameters.
 5. Click **Run**.
```
Comment (maycmlee) [inline content/en/security/application_security/threats/security_signals.md:67]: The numbers render correctly on the page, but just thought I'd point out that it's `1.` for the first two only.

File: content/en/service_management/workflows/trigger.md [hunk 2]
```diff
@@ -97,57 +100,65 @@ To test a monitor trigger:
...
 ## Trigger a workflow from incidents
 
-To trigger a workflow from incidents, create an incident notification rule:
-1. Create a workflow with a **Monitor, Incident, or Security signal** trigger, or add a **Monitor, Incident, or Security signal** trigger to an existing workflow.
-1. Click on the trigger in the workflow canvas and copy the **Mention handle**.
-1. From the [Incidents Settings][6] page, select **Rules**.
+To trigger a workflow from an incident notification rule, you must first add an incident trigger to your workflow:
+1. Add an incident trigger to your workflow:
+   - If your workflow doesn't have any triggers, click **Add Trigger** > **Incident**.
+   - If your workflow already has one more triggers and you're adding the security trigger as an additional trigger, click the **Add Trigger** (lightning bolt) icon and select **Incident**.
```
Comment (maycmlee) [inline content/en/service_management/workflows/trigger.md:147]:
```suggestion
   - If your workflow already has one or more triggers and you're adding the security trigger as an additional trigger, click the **Add Trigger** (lightning bolt) icon and select **Incident**.
```

**Review by maycmlee — APPROVED** [review]
Just some small suggestions!

File: content/en/security/application_security/threats/security_signals.md (hunk 1 above)
Comment (hestonhoffman) [reply content/en/security/application_security/threats/security_signals.md:67]: Oh good call, thanks!

**Pull request merged**